tests/telemetry/test_telemetry_metrics_e2e.py

```diff
@@ -33,6 +37,7 @@ def gunicorn_server(telemetry_metrics_enabled="true", token=None):
     env["_DD_TRACE_WRITER_ADDITIONAL_HEADERS"] = "X-Datadog-Test-Session-Token:{}".format(token)
     env["DD_TRACE_AGENT_URL"] = os.environ.get("DD_TRACE_AGENT_URL", "")
     env["DD_TRACE_DEBUG"] = "true"
+    env["DD_PATCH_MODULES"] = "flask:false"
```

Comment:
make sure we don't generate spans during these calls, or else the metrics get confusing, because we generate metrics for span creation now.